compiler-rt/test/builtins/Unit/extendhfxf2_test.c

```diff
@@ -0,0 +1,71 @@
+// RUN: %clang_builtins %s %librt -o %t && %run %t
+// REQUIRES: librt_has_extendhfxf2
```

Comment:
Not needed with the correct #if guard.

Reply:
Yes, it does seems redundant, but all the other tests had this. Should this be removed?

Reply:
I think this would be good